jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735RestrictionAppliesTo.java

```diff
@@ -0,0 +1,18 @@
+package us.dot.its.jpo.ode.plugin.j2735;
+
+public enum J2735RestrictionAppliesTo {
+	none,              
+    equippedTransit,  
+    equippedTaxis,
+    equippedOther,     
+    emissionCompliant, 
+    equippedBicycle,
+    weightCompliant,
+    heightCompliant,
+    pedestrians,
+    slowMovingPersons,
+    wheelchairUsers,
+    visualDisabilities,
+    audioDisabilities, 
+    otherUnknownDisabilities,
```

Comment:
i think in Java the last member of the enum should have a ";" or nothing (but not a ",")